charts/rancher-vsphere-cpi/100.6.0+up1.4.2/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: vSphere CPI
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.26.0-0'
```

Comment:
This chart is aimed at 2.6. Should the `kube-version` range here target only `< 1.25.0-0` instead?

Reply:
Thank you for catching this. Looks like there have been 2 patch bumps in the history https://github.com/rancher/charts/commits/dev-v2.6/packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch but the latest is incorrect. Looks like < 1.26 was what was pulled in from vsphere-charts. I'll manually update that to < 1.25 for this release line